contracts/collections/sg721-base/src/contract.rs

```diff
@@ -54,7 +56,8 @@ where
         self.parent.contract_info.save(deps.storage, &info)?;
 
         let minter = deps.api.addr_validate(&msg.minter)?;
-        self.parent.minter.save(deps.storage, &minter)?;
+        self.minter.save(deps.storage, &minter)?;
+        cw_ownable::initialize_owner(deps.storage, deps.api, Some(&msg.minter))?;
```

Comment:
So now the minter it stored in two places which is redundant. The owner should be the creator, but is already in `CollectionInfo`. I understand setting this is necessary per cw721 changes, but maybe we can refactor sg721 to include cw_ownable and remove redundancy. 

@larry0x Have any ideas here?

Reply:
> now the minter it stored in two places

Actually it's stored in 3 places?

1. in `CollectionInfo` as the creator
2. in `self.minter`
3. in `cw_ownable::OWNERSHIP`

Imo, (1) is ok - I consider the creator a separate concept from the minter. Consider a physical painting. The creator is the painter who painted it. Then there's the current owner of the painting which can be a separate person. Although in sg721 the creator and minter may always be the same but we can imagine in the future they may be allowed to be different accounts.

(2) - not sure how sg721 makes use of `self.minter`. If it uses cw-ownable to define who has minting power then `self.minter` may be redundant (and can be deleted during the migration).